blaze/compute/tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
```

Comment:
yeah, i did this so that `test_spark` and `test_sparksql` can share the instance. In fact, spark whines if you have more than one instance active at the same time.